packages/emile/emile.0.7/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name:         "emile"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/emile"
+bug-reports:  "https://github.com/dinosaure/emile/issues"
+dev-repo:     "git+https://github.com/dinosaure/emile.git"
+doc:          "https://dinosaure.github.io/emile/"
+license:      "MIT"
+synopsis:     "Parser of email address according RFC 822"
+description:  """Parser of email address according RFC 822"""
```

Comment:
Note that the `description` field is the same as the `synopsis`. If one is missing, the content is going to be taken from the other one so `description` can be removed if it's not going to be elaborated